src/index.js

```diff
@@ -40,9 +40,12 @@ module.exports = function({ template, transformFromAst, types }) {
 
         const { code } = transformFromAst(types.Program([path.node]));
 
-        const count = (code.match(/expect\(/g) || []).length;
-        const containsExpectAssertions = code.includes('expect.assertions(');
-        const containsHasAssertions = code.includes('expect.hasAssertions()');
+        // remove comments
+        const normalisedCode = code.replace(/\/\/(.*)/g, '').replace(/\/\*([\s\S]*?)\*\//g, '');
```

Comment:
Is the question mark (?) needed in the multi-line comment regex? (`([\s\S]*?)`)

Reply:
Yes, to avoid greediness, important when a statement is sandwiched between two comments like this:

```js
/* expect(...) */
expect(...);
/* expect(...) */
```

Reply:
Nice catch